packages/dd-trace/src/plugins/util/web.js

```diff
@@ -267,7 +267,7 @@ const web = {
       }
     }
 
-    const span = tracer.startSpan(name, { childOf, links: childOf?._links })
+    const span = tracer.startSpan(name, { childOf, links: childOf?._links, kind: SERVER })
```

Comment:
Is this just something we were missing before?

Reply:
No, we were just setting `span.kind` via `setTag`. But we need it at the time of span creation, so we gotta add it here.